CalibCalorimetry/EcalLaserAnalyzer/src/ME.cc

```diff
@@ -453,7 +453,7 @@ MEChannel* ME::regTree(int ireg) {
               leaf_ = leaf_->getDaughter(ieta, iphi, iLMRegion_);
               leaf_ = leaf_->getDaughter(ieta, iphi, iLMModule_);
               leaf_ = leaf_->getDaughter(ieta, iphi, iSuperCrystal_);
-              leaf_ = leaf_->getDaughter(ieta, iphi, iCrystal_);
+              leaf_->getDaughter(ieta, iphi, iCrystal_);
```

Comment:
I'm not sure I understand why line 456 was changed but not l453-455

Reply:
`getDaughter()` returns a new MEChannel if it has not been created yet. Since the crystal is the smallest unit it has to stop there and does not branch anymore.

Reply:
I see, thanks!